tmk_core/common/command.c

```diff
@@ -365,6 +365,7 @@ static bool command_common(uint8_t code) {
             stop_all_notes();
             shutdown_user();
 #else
+            shutdown_user();
```

Comment:
Cant leave it as a suggestion, but why not reuse the existing function as its defined in `quantum.h`?
```c
        case MAGIC_KC(MAGIC_KEY_BOOTLOADER):
        case MAGIC_KC(MAGIC_KEY_BOOTLOADER_ALT):
            print("\n\nJumping to bootloader... ");
            reset_keyboard();
            break;
```

Reply:
Not sure about the relationship between the codes under quantum and the original tmk_core, are there any restrictions to prevent call functions from quantum in the tmk_core layer?

Reply:
its already included at the top of the file <https://github.com/qmk/qmk_firmware/blob/b5fd7a606d3019cfb321b208fc7b06757549b500/tmk_core/common/command.c#L35>